content/en/tracing/security/agent_span_modification.md

```diff
@@ -0,0 +1,240 @@
+```
+
+[1]: /tracing/visualization/#spans
+{{% /tab %}}
+{{% tab "MemCached" %}}
+
+Applies to [spans][1] of type `memcached`, more specifically, to the `memcached.command` span tags:
+
+```yaml
+apm_config:
+  enabled: true
+
+  ## (...)
+
+  obfuscation:
+    memcached:
+      enabled: true
+```
+
+[1]: /tracing/visualization/#spans
+{{% /tab %}}
+{{% tab "Http" %}}
+
+HTTP obfuscation rules for `http.url` metadata in [spans][1] of type `http`:
+
+```yaml
+apm_config:
+  enabled: true
+
+  ## (...)
+
+  obfuscation:
+    http:
+      remove_query_string: true
+      remove_paths_with_digits: true
+```
+
+* `remove_query_string`: If true, obfuscates query strings in URLs.
+* `remove_paths_with_digits`: If true, path segments in URLs containing digits are replaced by "?".
+[1]: /tracing/visualization/#spans
+{{% /tab %}}
+{{% tab "Stack Traces" %}}
+
+Set the `remove_stack_traces` parameter to true, to remove stack traces and replace them with `?`.
+
+```yaml
+apm_config:
+  enabled: true
+
+  ## (...)
+
+  obfuscation:
+    remove_stack_traces: true
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Replace rules for tag filtering
+
+To scrub sensitive data from your [span][2]'s tags, use the `replace_tags` setting [in your datadog.yaml configuration file][3] or the `DD_APM_REPLACE_TAGS` environment variable. It is a list containing one or more groups of parameters that describe how to perform replacements of sensitive data within your tags. These parameters are:
+
+* `name`: The key of the tag to replace. To match all tags, use `*`. To match the resource, use `resource.name`.
+* `pattern`: The regexp pattern to match against.
+* `repl`: The replacement string.
+
+For example:
+
+{{< tabs >}}
+{{% tab "datadog.yaml" %}}
+
+```yaml
+apm_config:
+  replace_tags:
+    # Replace all characters starting at the `token/` string in the tag "http.url" with "?":
+    - name: "http.url"
+      pattern: "token/(.*)"
+      repl: "?"
+    # Replace all the occurrences of "foo" in any tag with "bar":
+    - name: "*"
+      pattern: "foo"
+      repl: "bar"
+    # Remove all "error.stack" tag's value.
+    - name: "error.stack"
+      pattern: "(?s).*"
+```
+
+{{% /tab %}}
+{{% tab "Environment Variable" %}}
+
+```shell
+DD_APM_REPLACE_TAGS=[
+      {
+        "name": "http.url",
+        "pattern": "token/(.*)",
+        "repl": "?"
+      },
+      {
+        "name": "*",
+        "pattern": "foo",
+        "repl": "bar"
+      },
+      {
+        "name": "error.stack",
+        "pattern": "(?s).*"
+      }
+]
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Resource filtering
+
+The Agent can be configured to exclude a specific resource from traces sent by the Agent to Datadog. To prevent the submission of specific resources, use the `ignore_resources` setting in the `datadog.yaml` file . This setting enables the creation of a list containing one or more regular expressions, which instructs the Agent to filter out traces based on their resource name.
```

Comment:
```suggestion
The Agent can be configured to exclude a specific resource from traces sent by the Agent to Datadog. To prevent the submission of specific resources, use the `ignore_resources` setting in the `datadog.yaml` file . Then create a list of one or more regular expressions, specifying which resources the Agent will filter out based on their resource name.
```